### PR TITLE
747 autofill instructions modal

### PR DESCRIPTION
**Pull request by joeyg**
Resolves part of #747. Connected to #747. Does need tests.

This PR sets up the instructions modal in Settings.
simulator screen shot - iphone 6s - 2018-09-28 at 12 31 48

simulator screen shot - iphone xs - 2018-09-28 at 12 31 56

### CODE REVIEW

File: lockbox-ios/View/AutofillInstructionsView.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        layer.frame = self.videoView.bounds
+        self.videoView.layer.addSublayer(layer)
+        player.play()
+
+        NotificationCenter.default.rx
+            .notification(NSNotification.Name.AVPlayerItemDidPlayToEndTime)
+            .subscribe({_ in
+                player.seek(to: CMTime.zero)
+                player.play()
+            }).disposed(by: self.disposeBag)
```
Comment (jhugman) [inline lockbox-ios/View/AutofillInstructionsView.swift]: Looking at this, I think I'd prefer the name of the method to mention that it's a video on a loop. e.g. `setUpVideoLoop()`.

File: lockbox-iosTests/AutofillInstructionsPresenterSpec.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+@testable import Lockbox
+
+class AutofillInstructionsPresenterSpec: QuickSpec {
+    class FakeAutofillInstructionsView: AutofillInstructionsViewProtocol {
+        let finishButtonTapStub = PublishSubject<Void>()
+
+        var finishButtonTapped: Observable<Void> {
+            return finishButtonTapStub.asObserver()
+        }
+
```
Comment (jhugman) [inline lockbox-iosTests/AutofillInstructionsPresenterSpec.swift]: Nit: whitespac.e

**Comment by changecourse** [comment]
@joeyg is this ready for UX review, presumably?

**Comment by joeyg** [comment]
@changecourse yep!

**Comment by joeyg** [comment]
@nickbrandt updated the PR with your notes.
simulator screen shot - iphone xs - 2018-10-02 at 16 15 58

**Comment by nickbrandt** [comment]
@joeyg the changes look great! Thank you for taking care of those items.

I realized I missed 1 small thing related to copy. Could you update the first instruction to:
Open *the* Settings app (instead of to)

After that we should be good!

**Comment by joeyg** [comment]
@nickbrandt fixed.

simulator screen shot - iphone xs - 2018-10-03 at 09 05 10

**Pull request merged**